Dockerfile.curl.tpl

```diff
@@ -0,0 +1,7 @@
+FROM #{FROM}
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+		ca-certificates \
+		curl \
+		wget \
+	&& rm -rf /var/lib/apt/lists/*
```

Comment:
Can you add a new line at the end of all these templates? It's just a linux convention